gui/marketBrowser.py

```diff
@@ -94,7 +110,7 @@ def toggleMetaButton(self, event):
             # Do 'nothing' if we're trying to turn last active button off
             if len(activeBtns) == 1 and activeBtns.pop() == ebtn:
                 # Keep button in the same state
-                ebtn.SetValue(True)

```

Comment:
Also tested on macOS Sierra. I'm starting to look into the copy/paste issue and the tooltips and verified figured I would verify this.